src/commonMain/kotlin/org/kosat/heuristics/PriorityQueue.kt

```diff
@@ -94,30 +96,33 @@ class PriorityQueue {
     }
 
     fun buildHeap(activity: MutableList<Double>) {
```

Comment:
I don't really like this implementation. The standard way of build a heap from an array is as simple as follows:
1. Allocate a dummy heap from the array. (`heap = array`, index/order/whatever structure use use for indexing/ordering = initially empty vec).
2. Iterate over elements of `heap` from the end(!). Sift_down (w.r.t. min-heap) each element.
3. ??? Done!